integration-tests/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "commerce-sdk-integration-test",
+  "version": "1.4.1-beta.0",
+  "description": "Salesforce Commerce Cloud SDK Integration Test",
+  "homepage": "https://developer.commercecloud.com/s/",
+  "scripts": {
+    "test": "nyc --check-coverage=false mocha \"test_integration/*.test.ts\" --retries 2 --timeout 5000 --reporter=xunit --reporter-options output=../reports/integration.xml",
+    "fetch:internal": "npm pack ../packages/core && npm pack ../packages/generator && npm install ",
+    "test:integration": "npm run fetch:internal &&  npm run test",
+    "test:integration:redis": "fetch:internal && npm run redis --prefix ../packages/core && npm run test; npm run redis:shutdown --prefix ../packages/core",
+    "test:integration:redis:debug": "fetch:internal && npm run redis --prefix ../packages/core && npm run test; npm run redis:shutdown:debug --prefix ../packages/core"
+  },
+  "dependencies": {
+    "@commerce-apps/core": "file:./commerce-apps-core-1.4.1-beta.0.tgz",
```

Comment:
How do these files get here?

Reply:
npm pack, done in the fetch:internal script, creates the them and drops them in the current working directory, in this case the integration-tests folder.